src/mongocxx/collection.hpp

```diff
@@ -624,7 +624,7 @@ MONGOCXX_INLINE bsoncxx::stdx::optional<result::insert_many> collection::insert_
     document_view_iterator_type end,
     const options::insert& options
 ) {
-    class bulk_write writes(false);
+    class bulk_write writes(options.ordered().value_or(true));
```

Comment:
Is the change in default behavior here intentional? If the user doesn't specify, we want to treat their updates as ordered? That seems reasonable. Is there a spec that mandates one choice or the other?